### PR TITLE
fixing clang-tidy errors from recent imtui change

### PR DESCRIPTION
**Pull request by katemonster33**
#### Summary
Bugfixes "Fixing a clang-tidy error from a recent ImTui-related PR"

#### Purpose of change
#73616 got merged with a clang-tidy error still outstanding. this fixes it

### CODE REVIEW

File: src/cata_imgui.cpp [hunk 1]
```diff
@@ -38,13 +38,13 @@ std::array<pairs, 100> colorpairs;   //storage for pair'ed colored
 
 std::vector<std::pair<int, ImTui::mouse_event>> imtui_events;
 
-int GetFallbackStrWidth( const char *s_begin, const char *s_end,
+static int GetFallbackStrWidth( const char *s_begin, const char *s_end,
                          const float scale )
```
Comment (Brambor) [inline src/cata_imgui.cpp:42]: This looks like it will need astyle.

File: src/cata_imgui.cpp [hunk 2]
```diff
@@ -212,18 +212,18 @@ cataimgui::client::client( const SDL_Renderer_Ptr &sdl_renderer, const SDL_Windo
...
 // this function QUEUES a character to be drawn
-bool CanRenderFallbackChar( ImWchar wch )
+static bool CanRenderFallbackChar( ImWchar wch )
 {
     return wch != 0;
 }
 
-int GetFallbackStrWidth( const char *s_begin, const char *s_end,
+static int GetFallbackStrWidth( const char *s_begin, const char *s_end,
                          const float scale )
```
Comment (Brambor) [inline src/cata_imgui.cpp:221]: Astyle as well

**Review by Brambor — COMMENTED** [review]
Astyle it.

File: src/cata_imgui.cpp (hunk 1 above)
Comment (katemonster33) [reply src/cata_imgui.cpp]: ah indeed, done

File: src/cata_imgui.cpp (hunk 2 above)
Comment (katemonster33) [reply src/cata_imgui.cpp]: also done

**Pull request merged**